js/start.js

```diff
@@ -626,6 +626,7 @@ function start() {
                 const parsedData = app.walletBalance.parse({
                   confirmed: e.jsonData.walletUpdate.confirmed,
                   unconfirmed: e.jsonData.walletUpdate.unconfirmed,
+                  height: e.jsonData.walletUpdate.height,
```

Comment:
Should this change be in this PR? It doesn't seem to be related to trumbowyg.

Reply:
Ah, that got merged in from master when I reverted start.js. I'll remove that...

Reply:
Please make sure it matches master. It may just be a situation where you need to merge master into your branch.

Reply:
Fixed.

Reply:
Please make sure it matches master. It may just be a situation where you need to merge master into your branch.